test/nope_test.go

```diff
@@ -46,7 +46,7 @@ func TestTouchSimple(t *testing.T) {
 		t.Errorf("could not create file: " + testFileName)
 	}
 	var touchCmd = "touch " + testFileName
-	testCommand(touchCmd, t)
+	assert.Equal(t, testCommand(touchCmd, t), "rm testfile.txt", "the 'touch' command should be correctly matched")
```

Comment:
can we please extract the `the X command should be correctly matched` into a helper func?